src/python_interactive_client/osrd_interactive.py

```diff
@@ -160,13 +161,15 @@ async def init(self, infra_path, rolling_stocks_path=None):
         return response
 
     async def create_simulation(
-        self, simulation_path, successions_path=None, breakpoints=[]
+        self, simulation_path, successions_path=None, breakpoints=[], reference_times={}
```

Comment:
there's a very nasty mistake here:
the instance of the default argument is shared between all calls, it's evaluated at function declaration time.
Please do this instead:
```python
breakpoints=None, reference_times={}):
if breakpoints is None:
    breakpoints = []
...
```